packages/mock-wrapper/src/components/RootPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+export const RootPage = ({ children }) => <div>{children}</div>;
+
+RootPage.propTypes = {
+  children: PropTypes.node
+};
+
+RootPage.displayName = "Root";
+
+export default connect(
+  () => ({}),
+  () => ({})
+)(RootPage);
```

Comment:
You can remove these two empty functions and just make it:
```javascript
export default connect()(RootPage);
```

Reply:
Info on [what's going on here](https://stackoverflow.com/questions/41044121/why-call-connect-function-without-mapstatetoprops-or-mapdispatchtoprops-as-args/41044583) if you're curious.